frontend_tests/puppeteer_tests/drafts.ts

```diff
@@ -40,7 +40,7 @@ async function create_stream_message_draft(page: Page): Promise<void> {
     await page.keyboard.press("KeyC");
     await page.waitForSelector("#stream-message", {visible: true});
     await common.fill_form(page, "form#send_message_form", {
-        stream_message_recipient_stream: "all",
+        stream_message_recipient_stream: "Verona",
```

Comment:
This puppeteer test commit fails in isolation. It'd be best to reorder this to be a prep commit that passes in isolation, and then having whatever commit is changing something that this test relies on change the test to verify the new behavior.

```
AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:
+ actual - expected

+ ''
- 'Updated stream message'
    at test_edited_draft_message (/home/tabbott/zulip/frontend_tests/puppeteer_tests/drafts.ts:163:12)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async drafts_test (/home/tabbott/zulip/frontend_tests/puppeteer_tests/drafts.ts:257:5)
    at async CommonUtils.run_test_async (/home/tabbott/zulip/frontend_tests/puppeteer_lib/common.ts:619:13) {
  generatedMessage: true,
  code: 'ERR_ASSERTION',
  actual: '',
  expected: 'Updated stream message',
  operator: 'strictEqual'
}

```